### ISSUE TITLE
missing double precision column type

### ISSUE THREAD

**Issue by tr8dr** [JetBrains/Exposed#358]
Most applications that use floating point no longer use Float, rather Double.   I noticed that Exposed *does not provide* a Double column type.   Whereas, if have a table such as:

```
CREATE Table foo (
    a .  double precision,
    b .  double precision
    ...
);
```
Would want to be able to do the following on the kolin side:
```
object Foo : Table("foo") {
    val a = double("a")
    val b = double("b")
    ...
}
```

I added this in a fork and submitted a pull request.  As authors of the library you may wish to implement this differently.   I need to get this out of my private repo and into the official so we can use this from gradle within our organization.   Thanks!

**Comment by NekoiNemo** [comment]
```
/// ==== Double column ====

class DoubleColumnType : ColumnType() {
    override fun sqlType(): String = "DOUBLE PRECISION"

    override fun valueFromDB(value: Any): Any = when (value) {
        is Double -> value
        is Number -> value.toDouble()
        else -> error("Unexpected value of type Double: $value of ${value::class.qualifiedName}")
    }
}

fun Table.double(name: String): Column<Double> = registerColumn(name, DoubleColumnType())
```

That's how i solved it until it is actually implemented in lib

**Comment by tr8dr** [comment]
Thanks!, I would prefer not to use my own fork, so that is helpful.

**Comment by NekoiNemo** [comment]
Sorry, noticed you already made PR

**Comment by tr8dr** [comment]
Your solve here is nice.  I will keep the PR, but will use your code in conjunction with the current release.  Hopefully they will get this implemented at some point.

**Comment by NekoiNemo** [comment]
It's just a bandaid that can be applied from inside a project, without altering the lib. Of course proper solution is to also modify the dialect...

**Comment by Tapac** [comment]
Fixed in master (thank for PR #354 by @tr8dr)
